CommentsListHandler.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'; 
+import { comments } from './comments.js';
+import CommentHandler from './CommentHandler.js'
+
+function CommentsListHandler(props){
+  const currentPostComments = comments.filter(comment => comment.postId === props.postId);
+  const commentsList = [];
+  currentPostComments.map(item => {
```

Comment:
you should return value in map function,
map callback shouldn`t  have side effects